apps/builder/app/builder/features/settings-panel/props-section/animation/animation-keyframes.tsx

```diff
@@ -176,43 +189,97 @@ export const Keyframes = ({
 }) => {
   const ids = useIds(["addKeyframe"]);
 
+  // To preserve focus on children swap
+  const keyRefs = useRef(
+    Array.from({ length: keyframes.length }, (_, index) => index)
```

Comment:
interesting one, I never thought one can make array like this vs Array.from(new Array(keyframes.length))